src/pages/Chat.vue

```diff
@@ -219,6 +227,43 @@ export default {
     }
   },
   computed: {
+    stackedMessages () {
```

Comment:
@schancel This needs to be done progressively. Currently we simply clone all messages on every change.